### ISSUE TITLE
Feedback for “Annotation View”

### ISSUE THREAD

**Issue by annfrant** [aidenlx/obsidian-zotero-docs#3]
Is there a way to include manually created notes? I mean the notes that I just typed in after using "Add Note" in Zotero rather than those created with comments or highlights in an attached PDF.

**Comment by aidenlx** [comment]
checkout latest beta version: https://zotlit.aidenlx.top/changelog/v1.1.0#v110-beta11---beta13

**Comment by annfrant** [comment]
This is fantastic! I have spent the last two days getting it working for my purposes. (Given that the documentation is -- understandably -- incomplete, it took me some time to get the syntax in the templates right.) I was mostly successful, but I can report a couple of small issues. I found work-arounds for all of them, but they are things that I would hope would get fixed for the release version: 

1. I could only use the Live Template Editor after attaching a PDF to the reference in Zotero. (I have been using Zoteros since long before it was possible to do nice highlighting in the program, so my PDFs all live elsewhere.) A dummy PDF did the job, but without it, I just got an error message that blocked all further progress. Once I had my templates working, I could delete the PDF again without preventing the creation of LiteratureNotes.
2. In order to facilitate refactoring my notes in Obsidian (or any other markdown program) I make the title of my Zotero notes a heading (##). Zotlit added two spaces before every ##, which I had to remove to be able to use Note Refactor in Obsidian. This behaviour is undesirable.
3. Zotlit also exported all of the notes in seemingly random order despite my careful use of headings that begin with page numbers to keep everything in order. The Sort & Permute plugin fixed this for me, but it would obviously be nice to have a way of keeping the order the same as in Zotero.

I would regard the list above as bugs. The following are more "nice to haves": 

1. If memory serves, I also had to remove language about annotations from the templates to make my non-PDF system work. I now can't remember if that was necessary or not, but it would still be nice to have a a clearer distinction between notes and annotations built into the whole system. Even just providing sample templates for the different scenarios -- notes only, annotations only, or both notes and annotations -- would be very helpful. Ideally, though, I would like to be able to use the Zotero plugin to already distinguish between these scenarios.
2. I am very much looking forward to being to export in batches!
3. It would be lovely to have an easy way of switching templates for different categories of note. For example, I am an historian, and my Obsidian templates for archival notes look quite different from my general research notes. I also write about films, which require very different meta. It isn't a big deal to do some cutting and pasting into the Zotlit templates, but yeah... would be nice to be able hit a switch of some kind. In the best of all possible worlds, Zotlit would choose the correct template based on the Zotero reference type.

I hope this helps you, and I am already very grateful for your work!
